autoload/lsp/omni.vim

```diff
@@ -303,7 +311,6 @@ function! lsp#omni#get_vim_completion_items(options) abort
     let l:start_characters = [] " The mapping of item specific start_character.
     let l:vim_complete_items = []
     for l:completion_item in l:items
-        let l:expandable = get(l:completion_item, 'insertTextFormat', 1) == 2
```

Comment:
Move to https://github.com/prabirshrestha/vim-lsp/blob/4823c1e3e240bffb25a83b384e9fbfe293e4ae50/autoload/lsp/omni.vim#L351 because `l:expandable` is not used util here.